models/mpas_atm/model_mod.f90

```diff
@@ -209,14 +209,15 @@ module model_mod
 integer, parameter :: TIMELEN = 19
 
 ! Real (physical) constants as defined exactly in MPAS.
-! redefined here for consistency with the model.
-real(r8), parameter :: rgas = 287.0_r8
-real(r8), parameter :: rv = 461.6_r8
-real(r8), parameter :: cp = 1003.0_r8
-real(r8), parameter :: cv = 716.0_r8
+! redefined here for consistency with the model (MPAS/src/framework/mpas_constants.F).
+real(r8), parameter :: rgas = 287.0_r8  ! = R_d (Gas constant for dry air [J kg-1 K-1])
+real(r8), parameter :: rv = 461.6_r8    ! = R_v (Gas constant for water varpor [J kg-1 K-1])
+real(r8), parameter :: cp = 7.*rgas/2.  ! = 1004.5
```

Comment:
Another comment might be helpful here: "For an ideal diatomic gas, cp:cv:R is 7:5:2"